### ISSUE TITLE
How to use the tydiqa dataset

### ISSUE THREAD

**Issue by marc-tanti** [google-research/xtreme#53]
I'm confused with this because it doesn't seem to have a test set and the paper doesn't say which other dataset was used as a test set. The dev set also does not seem to be labelled. Furthermore, I don't see how to use tydiqa-goldp-v1.1-train.json together with the other training set files. Can someone help please?

**Comment by sebastianruder** [comment]
Hi Marc, the Gold Passage version of TyDiQA does not have separate dev and test sets, so the dev sets are used for testing. This is also why they are not labelled as we remove their annotations.

When using the `download_data.sh` script, [this line](https://github.com/google-research/xtreme/blob/master/scripts/download_data.sh#L184) calls the `utils_preprocess.py` script, which is responsible for splitting the `tydiqa-goldp-v1.1-train.json` file, which consist data in all languages into language-specific files. If you check your `download` folder, you should find a `tydiqa-goldp-v1.1-train` folder within the `tydiqa` folder that contains the language-specific train files.

Let us know if you have any further questions.
